docs/layers/text-layer.md

```diff
@@ -113,6 +113,13 @@ The maximum size in pixels.
 
 If on, the text always faces camera. Otherwise the text faces up (z).
 
+##### `backgroundColor` (Array, optional) ![transition-enabled](https://img.shields.io/badge/transition-enabled-green.svg?style=flat-square")
+
+- Default `null`
+
+The color to use for text background, in `[r, g, b]`. Each component is in the `[0, 255]` range.
+Alpha is determined by the per-object color.
```

Comment:
"The alpha of the background matches the opacity of each object, controlled by the props `getColor` and `opacity`."